bitnami/mongodb/values.yaml

```diff
@@ -348,10 +365,13 @@ labels: {}
 ## Ignored when mongodb.architecture=standalone
 ##
 replicaCount: 2
-## @param strategyType StrategyType for MongoDB(&reg;) statefulset
-## It can be set to RollingUpdate or Recreate by default.
+## @param updateStrategy.type Deployment strategy type
```

Comment:
Not a deployment, since this may vary depending on `useStatefulset`, please mention it and reference both documentations.

Reply:
Done!